packages/esm-service-queues-app/src/metrics/metrics-cards/patients-seen.extension.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import MetricsCard from './metrics-card.component';
+import { useConfig } from '@openmrs/esm-framework';
+import { useActiveVisits } from '../metrics.resource';
+import { useSelectedService } from '../../helpers/helpers';
+import { useQueueEntries } from '../../hooks/useQueueEntries';
+import { type ConfigObject } from '../../config-schema';
+import { useUniquePatientsWithAppointmentsCount } from '../../hooks/useUniquePatientsWithAppointmentsCount';
+
+export default function PatientsSeenExtension() {
+  const { t } = useTranslation();
+  const { showPatientSeenCard } = useConfig<ConfigObject>();
+  const currentService = useSelectedService();
+  const { activeVisitsCount } = useActiveVisits();
+  const { count: todaysAppointmentsCount } = useUniquePatientsWithAppointmentsCount();
+
+  const unScheduledPatientsCount = Math.max(activeVisitsCount - todaysAppointmentsCount, 0);
+
+  const { queueEntries } = useQueueEntries({
+    service: currentService?.serviceUuid,
+    isEnded: false,
+  });
+
+  const inServiceCount = queueEntries?.filter((entry) => entry.status?.display === 'In Service')?.length || 0;
+
+  return (
+    <MetricsCard
+      label={t('patients', 'Patients')}
+      headerLabel={t('patientsSeen', 'Patients Seen')}
```

Comment:
```suggestion
      headerLabel={t('patientsSeen', 'Patients seen')}
```

To align with the other sentence case convention that other metric tile labels follow.